src/mscorlib/mscorlib.csproj

```diff
@@ -181,7 +181,8 @@
   </PropertyGroup>
 
   <Import Project="$(MSBuildThisFileDirectory)Tools\Versioning\GenerateVersionInfo.targets"/>
-  <Import Project="GenerateSplitStringResources.targets"/>
+  <Import Project="GenerateSplitStringResources.Windows.targets" Condition="'$(OsEnvironment)'=='Windows_NT'"/>
```

Comment:
I can split this on Open (Python) and closed (ResGen).  I don't like the other approach.  I would rather land the changes on GH first and the do the TFS specific work in TFS.